plugins/com.google.cloud.tools.eclipse.appengine.localserver/META-INF/MANIFEST.MF

```diff
@@ -16,4 +19,16 @@ Require-Bundle: org.eclipse.wst.server.core;bundle-version="1.7.0",
  org.eclipse.jst.common.project.facet.core;bundle-version="1.4.400",
  org.eclipse.jst.common.project.facet.ui;bundle-version="1.4.510",
  javax.servlet;bundle-version="3.1.0";visibility:=reexport,
```

Comment:
Re-exporting is generally not a good idea.  Why do you think this is necessary here?